core/DataTable/Filter/ColumnDelete.php

```diff
@@ -161,7 +161,7 @@ protected function removeColumnsFromTable(&$table)
             }
             foreach ($this->columnsToRemove as $column) {
 
-                if (!array_key_exists($column, $row)) {
+                if (!property_exists($column, $row)) {
```

Comment:
Could you maybe undo this change? We will investigate this as part of https://github.com/matomo-org/matomo/issues/15210

Reply:
Yes, done.